installer/templates/ecto/model_case.ex

```diff
@@ -17,6 +20,7 @@ defmodule <%= application_module %>.ModelCase do
       alias <%= application_module %>.Repo
       import Ecto.Model
       import Ecto.Query, only: [from: 2]
+      import unquote(__MODULE__)
```

Comment:
I think this pattern is confusing for people stared. Can't we just repeat the name of the case? :)